src/qt/utilitydialog.cpp

```diff
@@ -135,6 +137,19 @@ void HelpMessageDialog::on_okButton_accepted()
     close();
 }
 
+void HelpMessageDialog::setAboutWindowTitle(const NetworkStyle *networkStyle)
+{
+    QString aboutTitle = tr("About %1").arg(PACKAGE_NAME);
+    if ((networkStyle) && (Params().GetChainType() != ChainType::MAIN)) aboutTitle.append(" " + networkStyle->getTitleAddText());
+    setWindowTitle(aboutTitle);
+}
+
+void HelpMessageDialog::setChainTypeIconOnAboutLogo(const NetworkStyle *networkStyle)
+{
+    const QSize requiredSize(1024,1024);
```

Comment:
nit
```suggestion
    const QSize requiredSize(1024, 1024);
```

Reply:
Thanks for reviewing! I'll change it if I have to re-touch it.

Reply:
done!